email_handler.py

```diff
@@ -1000,6 +1036,13 @@ def handle_reply(envelope, msg: Message, rcpt_to: str) -> (bool, str):
         LOG.w(f"No contact with {reply_email} as reverse alias")
         return False, status.E502
 
+    contact_alias_check = Alias.get_by(email=contact.website_email)
```

Comment:
I think we allow a contact to be an alias. For ex, a user can send an email from his alias to another user's alias.

Reply:
You're right. I've removed this condition